smart_open/s3.py

```diff
@@ -539,22 +533,19 @@ def detach(self):
         raise io.UnsupportedOperation("detach() not supported")
 
     def write(self, b):
-        """Write the given bytes (binary string) to the S3 file.
+        """Write the given buffer (bytes, bytearray, memoryview or any buffer
+        interface implementation) to the S3 file.
```

Comment:
Can you please include a link to the appopriate standard library documentation?

Is it this one? https://docs.python.org/3/c-api/buffer.html

Reply:
Yes, that's the right one.